pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -734,6 +735,10 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
+			if goruntime.GOOS == "windows" {
```

Comment:
move this up into the `successConfig` init, and invert this to only set the field on `linux`, since that's the only OS that supports it

```
		SingleProcessOOMKill: func() *bool {
			if goruntime.GOOS == "linux" {
				return ptr.To(!kubeletutil.IsCgroup2UnifiedMode())
			}
			return nil
		}(),
```